test/e2e/storage/csi_volumes.go

```diff
@@ -132,6 +132,7 @@ func (h *hostpathCSIDriver) createStorageClassTest(node v1.Node) storageClassTes
 		claimSize:    "1Gi",
 		expectedSize: "1Gi",
 		nodeName:     node.Name,
+		attach:       true,
```

Comment:
Should not true be in quotes? Since it is map[string]string..

Reply:
```
type storageClassTest struct {
	name           string
	cloudProviders []string
	provisioner    string
	parameters     map[string]string
	claimSize      string
	expectedSize   string
	pvCheck        func(volume *v1.PersistentVolume) error
	nodeName       string
	attach         bool
	volumeMode     *v1.PersistentVolumeMode
}
```

Reply:
Got it, by some reason . I thought it was part of `parameters` map.